.travis.yml

```diff
@@ -1,23 +1,39 @@
 language: ruby
+
+before_install:
+  - rvm get head
+  - gem install bundler
+
 script: "bundle exec rake test"
 sudo: required
 rvm:
```

Comment:
Are these the ruby versions? If so, why aren't we including supported versions of ruby?

Reply:
> are these changes just what is coming from upstream? If so can you please update the PR description?

Yes, All changes from upstream(https://github.com/ActsAsParanoid/acts_as_paranoid/tree/v0.5.0.rc1).